src/main/java/com/redhat/red/build/finder/report/TargetReport.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2017 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.finder.report;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import com.redhat.red.build.finder.KojiBuild;
+
+import org.apache.commons.collections4.MultiValuedMap;
+import org.apache.commons.collections4.multimap.ArrayListValuedHashMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TargetReport extends Report {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TargetReport.class);
+
+    private Map<String, List<String>> targetMap;
+
+    private static String tagNameToProductName(String tagName) {
+        String prodName = tagName.replaceAll("([a-z\\-]+)([0-9\\.][0-9a-z\\.]+)?(.*)", "$1-$2-$3");
```

Comment:
This appears to be only used for logging ?

Reply:
I was experimenting with some way of pretty-printing the target names (into more human-readable names), but I didn't include it in the output.

Reply:
The report now prints what it thinks the product name is instead of the target name. It's not perfect, particularly if the target name doesn't follow the standard pattern of including a version in it.